fastapi/applications.py

```diff
@@ -125,6 +131,16 @@ def __init__(
         self.openapi_schema: Optional[Dict[str, Any]] = None
         self.setup()
 
+    @property
+    def lifetime_dependencies(self) -> Dict[Tuple[Callable[..., Any], Tuple[str]], Any]:
```

Comment:
Rather than repeating the same type annotation over and over, define an alias?

Reply:
That sounds like a good idea.

Reply:
Here is the alias I made, also the one referred to in https://github.com/tiangolo/fastapi/pull/3516#discussion_r667614049: 
https://github.com/tiangolo/fastapi/blob/cecd77fdd12ac58c66631e9abac25e8c5149eccc/fastapi/dependencies/models.py#L7-L11